docs/experimental-features/beaconing-detection.asciidoc

```diff
@@ -0,0 +1,50 @@
+[[beaconing-detection]]
+== Beaconing detection
+
+This feature provides an early warning system for command and control (C2) beaconing activity. It monitors network traffic for indicators of compromise (IOCs), and provides analytics to aid your threat hunting.
```

Comment:
Small grammar fix, but also: if the abbreviations "C2" and "IOCs" aren't used again on this page, do they need to be abbreviated at all?
```suggestion
This feature provides an early warning system for command and control (C2) beaconing activity. It monitors network traffic for indicators of compromise (IOCs) and provides analytics to aid your threat hunting.
```